### ISSUE TITLE
[ENHANCEMENT] Repeat Print should remember Filament Mappings

### ISSUE THREAD

**Issue by BrienKing** [prusa3d/Prusa-Firmware-Buddy#4190]
### Printer model

XL

### Describe the enhancement

Right now if your select "Reprint" and the last print re-mapped the filaments since the order of the filaments changed you have to remap the filaments again.

### Expected functionality

I want reprint to do exactly what it just did.  If I remap filaments it should save that information and just reprint.

**Comment by danopernis** [comment]
I can easily see how implementing this would annoy some other user who would want to actually do the toolmapping again on reprint. But perhaps we should at least default to previous toolmapping so the workflow is more smooth.

**Comment by BrienKing** [comment]
Yes, I don't mind if it goes to the Tool Mapping screen again, as long as my previous mappings are there.

**Comment by CZDanol** [comment]
Internal ticket: BFW-6084

**Comment by o-vesely** [comment]
Hey just found this randomly, this has been finaly done and will be in fw 6.9 onward
It defaults to the previous (does not skip the mapping entirely)
